### ISSUE TITLE
Unstable run/start/stop behaviour

### ISSUE THREAD

**Issue by stackErr-NameNotResolved** [netzfisch/rpi-vpn-server#3]
tldr; step 1 and mostly step 4 have incorrect behaviour.

Step 1:
docker run --name vpnserver \
             --env HOSTNAME=your.domain.com \
             --env VPN_USER=name \
             --env VPN_PASSWORD=secret \
             --cap-add NET_ADMIN \
             --publish 500:500/udp \
             --publish 4500:4500/udp \
             --volume /vpn-secrets:/mnt \
             netzfisch/rpi-vpn-server

omitted --restart unless-stopped and --detach for debug purposes.

Expected: up and running vpn container.
Actual result: container shutsdown with "starter is already running (/var/run/starter.charon.pid exists) -- no fork done" error.

Step 2:
docker start vpnserver

Expected: up and running vpn container.
Actual result: up and running vpn container. (Tested by connecting with Android client)

Step 3:
docker stop vpnserver

Expected: stopped vpn container.
Actual result: stopped vpn container.

Step 4:
docker start vpnserver

Expected: up and running vpn container.
Actual result: container shutsdown with "starter is already running (/var/run/starter.charon.pid exists) -- no fork done" error.

Even after rebooting the PI, every "docker start vpnserver" will give "starter is already running (/var/run/starter.charon.pid exists) -- no fork done" error. Therefore it is impossible to really use this docker image.

**Comment by stackErr-NameNotResolved** [comment]
Any updates on this?

**Comment by netzfisch** [comment]
**tldr; works as intended, if you use "--restart unless-stopped"**

### Explanation

#### Step 1

Container has to restart after changing certificates + routes.

#### Step 4

After applying routes again - container stops and does not start again, cause "--restart unless-stopped" was omitted!

#### Add-on

Make sure **container can write to mounted volume**, otherwise described behavior could also happen!
